docs/api.rst

```diff
@@ -1518,6 +1518,28 @@ of :class:`enum.Enum`.
 
         See also :meth:`InteractionResponse.edit_message`
 
+.. class:: MessageInteraction
+
+    Represents the interaction data of a message if applicable
+
+    .. versionadded:: 2.0
+
+        Represents the raw data of the message interaction
+    .. attribute:: data
+
+        Represent the id of the message where the interaction occured
+    .. attribute:: id
+
+        Represents the type of message interaction
+    .. attribute:: type
+
+        Represents the name of the message interaction
+    .. attribute:: name
+
+        Represents the user that used the interaction
+        Or the member that used the interaction if guild applicable
+    .. attribute:: user
+
```

Comment:
```suggestion
    :members:
```

Reply:
What is the suggested change?

Reply:
It seems you just removed most of the attrs, but thanks for reminding me to update that last part